src/drive/styles/filelist.styl

```diff
@@ -340,6 +340,10 @@ column-width-thumbnail-bigger = 7rem
         button
             background silver
             opacity 1
+
+.fil-content-body
+    padding-bottom calc(56px + 4rem) // Claudy Fab size + margins around
+
```

Comment:
On main view, there was no problem because we add the same padding when the add button is displayed. But on recent, sharing, trash view there hasn't so I put counter padding without cancelling the add button padding : https://github.com/cozy/cozy-drive/pull/2891/files#diff-ba9ba0dfee2d21cbed24e34c5758dbf04700a03dda70b5af3d2038ae4eaf9a63R359